CHANGES

```diff
@@ -134,6 +134,14 @@ Display diffs between proof steps
   details.  Showing diffs in Proof General requires small changes to PG
   (under discussion).
 
+Notations
+
+- Added [++] infix for [VectorDef.append].
```

Comment:
Is there any reason to use here `[` ... `]` and not `` ` `` ... `` ` `` like in the rest of the paragraph? 

[Note that there is an ongoing [discussion](https://github.com/coq/coq/issues/8180) on the style to use for  `CHANGES` and, even if w/o final conclusion yet, it seems that we are going towards adopting markdown for `CHANGES`.]

Reply:
I realized there are two styles after the first commit. Maybe I should unify into ``` ` ``` style.